File: solutions/001/pe_001.py
MAX = 1000


def pe_001():
    answer = 0
    count = 1
    while count < MAX:
        if count % 3 == 0 or count % 5 == 0:
            answer += count
        count += 1
    return answer


if __name__ == '__main__':
    print(pe_001())

File: solutions/001/pe_001.c
#include <stdio.h>

int main(void)
{
    int answer = 0;

    for (int i = 0; i < 1000; i++)
    {
        if (i % 3 == 0 || i % 5 == 0)
        {
            answer += i;
        }
    }

    printf("%i\n", answer);

    return 0;
}
